internal/app/app.go

```diff
@@ -37,7 +37,7 @@ type githubClient interface {
 	GetLatestCommitFromBranch(ctx context.Context, owner, repo, branch string) (string, error)
```

Comment:
Ensure to handle `clients.Side` types safely in `AddCommentToPullRequestReview` method to prevent potential runtime panics or issues arising from passing invalid or nil values.